File: app/src/main/java/com/example/rapidchat/entities/Usuario.java
package com.example.rapidchat.entities;

public class Usuario {
    private int numeroTelefono = 0;
    private String nombre = "";
    private String apellidos = "";

    public Usuario(int numeroTelefono, String nombre, String apellidos) {
        String strNumeroTelefono = numeroTelefono + "";
        if (strNumeroTelefono.length() == 9) {
            this.numeroTelefono = numeroTelefono;
        }
        if (!nombre.equals("") && nombre != null) {
            this.nombre = nombre;
        }
        if (!apellidos.equals("") && apellidos != null) {
            this.nombre = apellidos;
        }
    }

    public int getNumeroTelefono() {
        return numeroTelefono;
    }

    public void setNumeroTelefono(int numeroTelefono) {
        String strNumeroTelefono = numeroTelefono + "";
        if (strNumeroTelefono.length() == 9) {
            this.numeroTelefono = numeroTelefono;
        }
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        if (!nombre.equals("") && nombre != null) {
            this.nombre = nombre;
        }
    }

    public String getApellidos() {
        return apellidos;
    }

    public void setApellidos(String apellidos) {
        if (!apellidos.equals("") && apellidos != null) {
            this.nombre = apellidos;
        }
    }

}

File: app/src/main/java/com/example/rapidchat/entities/Usuario.kt
package com.example.rapidchat.entities

class Usuario(numeroTelefono: Int, nombre: String?, apellidos: String?) {
    private var numeroTelefono = 0
    private var nombre = ""
    private var apellidos = ""

    init {
        if (numeroTelefono.toString().length == 9) {
            this.numeroTelefono = numeroTelefono
        }
        if (nombre != "" && nombre != null) {
            this.nombre = nombre
        }
        if (apellidos != "" && apellidos != null) {
            this.nombre = apellidos
        }
    }

    fun getNumeroTelefono(): Int {
        return numeroTelefono
    }

    fun setNumeroTelefono(numeroTelefono: Int) {
        if (numeroTelefono.toString().length == 9) {
            this.numeroTelefono = numeroTelefono
        }
    }

    fun getNombre(): String {
        return nombre
    }

    fun setNombre(nombre: String?) {
        if (nombre != "" && nombre != null) {
            this.nombre = nombre
        }
    }

    fun getApellidos(): String {
        return apellidos
    }

    fun setApellidos(apellidos: String?) {
        if (apellidos != "" && apellidos != null) {
            this.apellidos = apellidos
        }
    }

}